src/server/config.rs

```diff
@@ -28,6 +28,9 @@ const DEFAULT_RECV_BUFFER_SIZE: usize = 128 * 1024;
 pub struct Config {
     pub cluster_id: u64,
 
+    // Server tags.
```

Comment:
I think it should be explained in user manual instead of here, because TiKV knows nothing about what the tags mean, just some stings to report to PD.